### PR TITLE
[#40] [UI] As a user, I can see a choice question

### LINKED ISSUE #40: [UI] As a user, I can see a choice question
## Why
There are two types of Choice question; Single-choice question and Multiple-choice question. Choice questions allow users to pick the most suitable option to answer since the mobile app will lay out all the options.
As opposed to dropdown questions, Single-choice questions tend to be used when there are only a few options, for example, where do you hear out about us, how do you commute, etc.

## Acceptance Criteria
For choice questions,
- Display choices with checkboxes
- Highlight a selected choice by ticking its checkbox
- Reduce the opacity of non-selected choices and untick their checkbox
**DO NOT limit the maximum number of line on the Choice Text labels**

Image

## Design
[Choice Question](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge-Mobile-App?page-id=0%3A1&node-id=34%3A1172&scaling=min-zoom)

### PR DESCRIPTION
**Pull request by kaungkhantsoe**
Closes #40 

## What happened 👀

There are two types of Choice question; Single-choice question and Multiple-choice question. Choice questions allow users to pick the most suitable option to answer since the mobile app will lay out all the options.
As opposed to dropdown questions, Single-choice questions tend to be used when there are only a few options, for example, where do you hear out about us, how do you commute, etc.

## Insight 📝

- Display choices with checkboxes
- Highlight a selected choice by ticking its checkbox
- Reduce the opacity of non-selected choices and untick their checkbox
- DO NOT limit the maximum number of line on the Choice Text labels

## Proof Of Work 📹
Screenshot_20221003_115914

### CODE REVIEW

**Comment by ryan-conway** [comment]
@kaungkhantsoe How will we handle cases where there are more questions than available screen space? For example, this long screen has 20 choices but only 15 are displayed 🤔 This is assuming there are no other items on the screen like title or toolbar 🤣 

Screenshot 2565-10-17 at 10 41 00

File: app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyChoiceQuestionScreen.kt [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.foundation.BorderStroke
```
Comment (AVI5HEK) [inline app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyChoiceQuestionScreen.kt:3]: Not sure if this setting is updted on your AS.
image
image

File: app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyChoiceQuestionScreen.kt [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.kks.nimblesurveyjetpackcompose.R
+import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
+import com.kks.nimblesurveyjetpackcompose.ui.theme.NeuzeitFamily
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White50
+
+@Composable
+fun SurveyChoiceQuestionScreen(answers: List<SurveyAnswer>, onChooseAnswer: (answers: List<SurveyAnswer>) -> Unit) {
+    var selectedIndexList by remember { mutableStateOf(emptyList<Int>()) }
```
Comment (AVI5HEK) [inline app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyChoiceQuestionScreen.kt]:
```suggestion
    var selectedIndices by remember { mutableStateOf(emptyList<Int>()) }
```

**Review by AVI5HEK — APPROVED** [review]
LGTM

**Comment by kaungkhantsoe** [comment]
> @kaungkhantsoe How will we handle cases where there are more questions than available screen space? For example, this long screen has 20 choices but only 15 are displayed 🤔 This is assuming there are no other items on the screen like title or toolbar 🤣
> 
> Screenshot 2565-10-17 at 10 41 00

@ryan-conway I tired adding 20 items like you said and it shows like following but it was not scrollable. So I will be changing the type of layout from `Column` to `LazyColumn`.
Screenshot_20221017_150231

**Review by ryan-conway — APPROVED** [review]

**Pull request merged**